app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontAddProductsWithCustomOptionToWishlistTest.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontAddProductsWithCustomOptionToWishlistTest">
+        <annotations>
+            <features value="Wishlist"/>
+            <stories value="Add product with customizable option to wishlist"/>
+            <title value="Add product with customizable option to wishlist."/>
+            <description value="Add product with custom radio button option to wishlist."/>
+            <group value="wishlist"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-34245"/>
+        </annotations>
+        <before>
+            <createData entity="SimpleSubCategory" stepKey="categoryFirst"/>
+            <createData entity="SimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="categoryFirst"/>
+            </createData>
+
+            <updateData entity="productWithOptionRadiobutton" createDataKey="simpleProduct" stepKey="updateProductWithOptions">
+                <requiredEntity createDataKey="simpleProduct"/>
+            </updateData>
+
+            <createData entity="Simple_US_Customer" stepKey="customer"/>
+            <magentoCLI command="indexer:reindex" stepKey="reindex"/>
+            <magentoCLI command="cache:flush" stepKey="flushCache"/>
+        </before>
+        <after>
+            <deleteData createDataKey="simpleProduct" stepKey="deletesimpleProduct"/>
+            <deleteData createDataKey="categoryFirst" stepKey="deleteCategoryFirst"/>
+            <magentoCLI command="indexer:reindex" stepKey="reindex"/>
```

Comment:
Please, use the new non-blocking way of reindexing that has been recently introduced into MFTF

https://github.com/magento/magento2/blob/1e8ed89642906a7e20684da2f63cf72f149b52f8/app/code/Magento/Bundle/Test/Mftf/Test/AdvanceCatalogSearchBundleByPriceTest.xml#L49